configure.ac

```diff
@@ -118,6 +118,8 @@ AM_CONDITIONAL(ENABLE_ROLLBACK, [test "x$enable_rollback" = "xyes"])
 
 if test "x$enable_rollback" = "xyes"; then
 	AC_DEFINE(ENABLE_ROLLBACK, 1, [Enable rollback support])
+        # libmount is needed for rollback feature
+        AC_CHECK_HEADER(libmount/libmount.h,[],[AC_MSG_ERROR([Cannout find libmount headers. Please install libmount-devel])])
```

Comment:
Not all distributions call packages with header files "-devel", e.g. Debian calls them "-dev".